# Vulnerability Report for getwilds/scanpy:1.10.2

Report generated on Tue Apr  1 06:21:03 UTC 2025

    ...Pulling
    âœ— Pull failed
[31mERROR  [0m Status: failed to get image getwilds/scanpy:1.10.2: failed to pull image getwilds/scanpy:1.10.2: GET https://index.docker.io/v2/getwilds/scanpy/manifests/1.10.2: TOOMANYREQUESTS: You have reached your pull rate limit as 'tfirmanfh': dckr_jti_zUeDUeyUZ70liWOMpTY1KWFiYac=. You may increase the limit by upgrading. https://www.docker.com/increase-rate-limit, Code: 1 
Error generating all vulnerabilities report. Please try again later.
